docs/concepts/controller-manager.md

```diff
@@ -107,6 +107,27 @@ The `--event-ttl` should be larger than the `ttlNonShootEvents` or this controll
 
 Consequently, to ensure that `ExposureClass`s in-use are always present in the system until the last referring `Shoot` gets deleted, the controller adds a finalizer which is only released when there is no `Shoot` referencing the `ExposureClass` anymore.
 
+### [`ManagedSeedSet` Controller](../../pkg/controllermanager/controller/managedseedset)
+
+`ManagedSeedSet` object maintains a stable set of replica of `ManagedSeed`s i.e it gurantee the availability of a specified number of identical `ManagedSeed`s on an equal number of identical `Shoot`s.
+`ManagedSeedSet` controller creates and deletes `ManagedSeed`s and `Shoot`s in response to changes to the replicas and selector fields. For more info on `ManagedSeedSet` resource [refer](../../docs/proposals/13-automated-seed-management.md#managedseedsets).
+
+#### "Actuator"
```

Comment:
If we will mention that this reconciler uses an actuator, perhaps it makes sense to mention that there is an `Actuator` interface for which the default implementation does the steps described below. Similar to https://github.com/gardener/gardener/blob/master/docs/extensions/infrastructure.md#actuator-interface.
Although, this is more-or-less an implementation detail, so not sure if the `Actuator` struct/interface should be mentioned here and perhaps only leave the steps listed below?